sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/ScheduledReporterFactory.java

```diff
@@ -0,0 +1,48 @@
+package com.azure.cosmos.benchmark;
```

Comment:
code style: please add the header file to the new file.

same for all other new files.